RedHat-UX/red-hat-design-system
# feat(card): promo variant

## What I did

1. Added a new demo + styles for `<rh-card>` to be used for an "Inline Promo".
1. Fixed an issue with the `.empty` class not having enough specificity
1. Fixed a spacing issue with the `#body` when the `#footer` is `.empty`

## Testing Instructions

1. Review the [Inline Promo Issue](https://github.com/RedHat-UX/red-hat-design-system/issues/566)
1. Check the Deploy Preview for [`<rh-card>`'s Promo demo](https://deploy-preview-1569--red-hat-design-system.netlify.app/elements/card/demo/promo/)
    * [Promo](https://deploy-preview-1569--red-hat-design-system.netlify.app/elements/card/demo/promo/)
    * [Full Width Promo ](https://deploy-preview-1569--red-hat-design-system.netlify.app/elements/card/demo/promo-full-width/)
    * [Standard](https://deploy-preview-1569--red-hat-design-system.netlify.app/elements/card/demo/promo-standard/)
    * [Narrow](https://deploy-preview-1569--red-hat-design-system.netlify.app/elements/card/demo/promo-narrow/)

## Notes to Reviewers

[Benny thinks](https://github.com/RedHat-UX/red-hat-design-system/issues/566#issuecomment-2106269997) that Inline Promo should be a `<rh-card>` variant. Take a look at what I've done here and see if you agree.

### Questions:

1. Who should we show this to? Who will take on that task? ("...shop it around to studio devs." -Benny)

Related to #566
